docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,717 @@
+************************************************
+Track and Trade Transaction Family Specification
+************************************************
+
+Overview
+========
+
+The Track and Trade (T&T) transaction family allows users to track
+goods as they move through a supply chain. Records for goods include a
+history of ownership and custodianship, as well as histories for a
+variety of properties such a temperature and location. These
+properties are managed through a user-specifiable system of record
+types.
+
+
+State
+=====
+
+All T&T objects are serialized using Protocol Buffers before being
+stored in state. These objects include: Agents, Properties
+(accompanied by their auxiliary PropertyPage objects), Proposals,
+Records, and RecordTypes. As described in the Addressing section
```

Comment:
Can you make "Addressing" into a link?  There may be a better way, but I would add an anchor above the Addressing section, thusly: 

          .. _Addressing:
          
          Addressing
          ----------

Then, in the above sentence, I would replace the word "Addressing" with a references to the anchor:

          :ref:``Addressing``: 

NOTE:  There's only one backtick around Addressing.  Git won't let me show only one - it's either none or too many.

Reply:
P.S.  For a reference to something in the same file, I think you can omit the ":ref" and backticks, and simply add an underscore, thusly: 

          Addressing_

Reply:
I believe headers are automatically anchors, so you can just add the underscore to this line.